modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderUploadCSV.testcase

```diff
@@ -32,6 +32,111 @@ definition {
+		property portal.acceptance = "true";
+
+		task ("Given the buyer is importing the CSV order files") {
+			User.logoutPG();
+
+			CommerceLogin.miniumLogin(
+				password = "test",
+				urlAppend = "web/minium",
+				userEmailAddress = "buyer@liferay.com");
+
+			CommerceAcceleratorsInitializer.createNewOrderFromPendingOrders();
+
+			CommerceOrders.importOrderFromType(orderType = "CSV");
+
+			UploadDependencyFile.uploadFile(fileName = "commerce_csv_pagination.csv");
+
+			Button.clickImport();
+		}
+
+		task ("And can see data set is filled with the following data in the preview modal and only twenty items are displayed") {
+			Pagination.changePagination(itemsPerPage = "20 items");
+
+			CommerceEntry.viewPaginationResults(results = "Showing 1 to 20 of 216");
+
+			CommerceOrders.assertOrderImportPreviewEntry(
+				importStatus = "OK",
+				productName = "Brake Fluid",
+				productQuantity = 2,
+				productSku = "MIN93016A",
+				totalPrice = "160.00",
+				unitPrice = "80.00");
+
+			CommerceOrders.assertOrderImportPreviewEntry(
+				importStatus = "OK",
+				productName = "ABS Sensor",
+				productQuantity = 2,
+				productSku = "MIN93015",
+				totalPrice = "100.00",
+				unitPrice = "50.00");
+
+			CommerceOrders.assertOrderImportPreviewEntry(
+				importStatus = "OK",
+				productName = "Master Cylinder",
+				productQuantity = 2,
+				productSku = "MIN93026",
+				totalPrice = "78.00",
+				unitPrice = "39.00");
+		}
+
+		task ("When the user go to next page") {
+			Click(locator1 = "ACUtils#NEXT_PAGES");
+		}
+
+		task ("Then twenty different items are displayed") {
+			CommerceEntry.viewPaginationResults(results = "Showing 21 to 40 of 216");
+
+			CommerceOrders.assertOrderImportPreviewEntry(
+				importStatus = "OK",
+				productName = "Drive Shafts",
+				productQuantity = 2,
+				productSku = "MIN55855",
+				totalPrice = "792.00",
+				unitPrice = "396.00");
+
+			CommerceOrders.assertOrderImportPreviewEntry(
+				importStatus = "OK",
+				productName = "CV Axles",
+				productQuantity = 2,
+				productSku = "MIN55853",
+				totalPrice = "178.00",
+				unitPrice = "89.00");
+
+			CommerceOrders.assertOrderImportPreviewEntry(
+				importStatus = "OK",
+				productName = "Mount",
+				productQuantity = 2,
+				productSku = "MIN55857",
+				totalPrice = "6.00",
+				unitPrice = "3.00");
+
+			AssertElementNotPresent(
+				key_columnNumber = 1,
+				key_productName = "Brake Fluid",
+				key_value = "MIN93016A",
+				locator1 = "CommerceEntry#TABLE_SEARCH_BY_NAME_AND_COLUMN_CONTENT");
+
+			AssertElementNotPresent(
+				key_columnNumber = 1,
+				key_productName = "ABS Sensor",
+				key_value = "MIN93015",
+				locator1 = "CommerceEntry#TABLE_SEARCH_BY_NAME_AND_COLUMN_CONTENT");
+
+			AssertElementNotPresent(
+				key_columnNumber = 1,
+				key_productName = "Master Cylinder",
+				key_value = "MIN93026",
+				locator1 = "CommerceEntry#TABLE_SEARCH_BY_NAME_AND_COLUMN_CONTENT");
+
+			IFrame.closeFrame();
+		}
```

Comment:
The assertion of the previous page arrow is missing. Also paginating by directly clicking on a page number is missing